deploy/operator.yaml

```
@@ -129,6 +133,43 @@ spec:
             limits:
               memory: 256Mi
               cpu: 500m
+        - args:
+            - --enable-leader-election
```

```suggestion
            - --enable-leader-election
            - --metrics-addr
            - "0"
```

Switch off metrics collection in DWCO, otherwise it enters a crash loop.